packages/core/src/processed-data-manager.ts

```diff
@@ -0,0 +1,134 @@
+import get from 'lodash/get';
+import { ApplicationError } from 'n8n-workflow';
+import type {
+	IProcessedDataManager,
+	ICheckProcessedOptions,
+	ICheckProcessedOutput,
+	ICheckProcessedOutputItems,
+	IDataObject,
+	ProcessedDataContext,
+	ProcessedDataItemTypes,
+	ICheckProcessedContextData,
+} from 'n8n-workflow';
+
+export class ProcessedDataManager {
```

Comment:
Could we sort documentation of what this is and what it does?

Reply:
Added some description for the class